con-master/src/main/java/ba/codecentric/master/controller/RoomTypeController.java

```diff
@@ -28,11 +30,20 @@ public RoomTypeController(RoomTypeService roomTypeService, ImageService imageSer
     }
 
     @RequestMapping(value = "/room-type", method = RequestMethod.POST, consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
-    public RoomType saveRoom(@RequestPart("image") MultipartFile image, @Valid @RequestPart("roomType") RoomType roomType) throws IOException {
-        String fileName = imageService.saveImage(image.getInputStream(), image.getOriginalFilename());
-        if (fileName != null) {
-            roomType.setImage(fileName);
-            return roomTypeService.saveRoom(roomType);
+    public RoomType saveRoom(@RequestPart("image") MultipartFile image, @Valid @RequestPart("roomType") RoomType roomType) {
```

Comment:
It is not part of this ticket, but we shouldn't have too much logic inside controllers. Also, it should be refactored. In this one method we have too many different functionalities.